### PR TITLE
fix ens name turns into public key while editing message

### LINKED ISSUE #14051: ENS name turns into public key after editing message containing mention with this ENS
**Steps:**

1. Open any chat with member having ENS name
2. Send some message containing ENS name
3. Edit the sent message
4. Pay attention at ENS name in edited message

**Actual result:** private key and part of 3 random name is displayed instead of ENS name. 

**Expected result:** ENS name should be preserved in edited message

https://user-images.githubusercontent.com/97245802/191309513-1f5b8587-51b5-405f-9a2b-bf5fffd4161a.mp4

[geth.log](https://github.com/status-im/status-mobile/files/9609444/geth.log)
[Status.log](https://github.com/status-im/status-mobile/files/9609445/Status.log)
me

- Status version: nightly 
- Operating System: Android, iOS

### PR DESCRIPTION
**Pull request by Parveshdhull**
fixes https://github.com/status-im/status-mobile/issues/14051

## Issue Cause
1.` :ens-verified` param was missing, so the 3 word name is used instead of the ens name **_(bug 1)_**
2. we were setting original string with the public key as text input instead of string with ens resolved _**(bug 2)**_
3. For ens names we were appending extra @ (so two @'s total) (not actually bug)

### Testing Request:
Please test editing message in all chat types with and without ens name/mention

status: ready

### CODE REVIEW

**Review by audriu — APPROVED** [review]

**Review by flexsurfer — APPROVED** [review]

File: VERSION [hunk 1]
```diff
@@ -1 +1 @@
-1.20.0
+1.20.2
```
Comment (Parveshdhull) [inline VERSION:1]: Current [nightly builds](https://webapp.diawi.com/install/QeeccU) are still using the `1.20.0` as app version while the [current version](https://github.com/status-im/status-mobile/releases) of the app is `1.20.2`.
(I guess it is a small change, so separate PR is not required?)

cc: @flexsurfer

**Comment by pavloburykh** [comment]
@Parveshdhull thank you for the PR. Unfortunately, I am still reproducing the bug on my mainnet ens names (pavlo.stateofus.eth, pavlotest.eth, ditest.stateofus.eth). The initial problem is that ENS names are often not resolved after sending message containing this ENS name. The ENS turns into 3 random name. And in this case, after editing such message, 3 random name turns into public key. 

### ISSUE 1 Ens names are still turning into public key after editing message that contains mention with these ens names in case if ENS name is not resolved (turns into 3 random name) after sending a message

**Steps:**

1. Login multiaccount
2. Switch to Mainnet if needed
3. Start any kind of chat with @pavlo (or pavlotest.eth, ditest.stateofus.eth)
5. Mention @pavlo user and send message with mention
6. Observe if ENS name is turned into 3 random name after sending a message. If it happened and ENS name is not resolved - proceed to next Step. 
7. Edit message
8. Observe the result


**Actual result:** 
1. ENS name turns into 3 random name after sending message with mention (Step 6). It is not reproducible every time. Sometimes ENS is resolved successfully and in this case, editing message with resolved ENS mention does not lead to the bug of turning into public key. 
2. After editing message - 3 random name turns into public key. 
3. ENS name is also not resolved for other users who open profile page of ENS owner (3 random name is displayed instead). At the same time, ENS is displayed for it's owner on his profile page.  

**Expected result**: 
1. ENS name should not turn into 3 random name after sending message that contains mention with this ENS.
2. 3 random name should not turn into public key after editing message with mention.  

https://user-images.githubusercontent.com/97245802/195391344-1750f8fb-dca6-4f39-9fd9-be7b6d751a7a.mp4

@Parveshdhull If you consider this case not related to the initial issue - let me know and I will log a separate issue for that. Ping me if you'll need any help with reproduction the bug. 

Logs from the device, where ENS is not resolved are attached below. Let me know if you will need logs from the account of ENS owner. 

[geth.log](https://github.com/status-im/status-mobile/files/9766800/geth.log)
[Status.log](https://github.com/status-im/status-mobile/files/9766802/Status.log)

**Comment by pavloburykh** [comment]
### UPDATE:
ISSUE 1 is also reproducible for @qoqobolo 
Thank you for help, Diana.

**Comment by pavloburykh** [comment]
@Parveshdhull Thank you for your great work on the PR! Tested and ready for merge.

**Comment by Parveshdhull** [comment]
Thank you @pavloburykh and @qoqobolo for all your help in reproducing and fixing this issue.

**Pull request merged**